docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-kubernetes.md

```diff
@@ -77,13 +77,13 @@ Place the install script anywhere on each node, and name it `install.sh`.
 Install K3s on each server:
 
 ```
-INSTALL_K3S_SKIP_DOWNLOAD=true ./install.sh
+INSTALL_K3S_SKIP_DOWNLOAD=true INSTALL_K3S_VERSION=<VERSION> ./install.sh
 ```
 
 Install K3s on each agent:
 
 ```
-INSTALL_K3S_SKIP_DOWNLOAD=true K3S_URL=https://myserver:6443 K3S_TOKEN=mynodetoken ./install.sh
+INSTALL_K3S_SKIP_DOWNLOAD=true INSTALL_K3S_VERSION=<VERSION> K3S_URL=https://myserver:6443 K3S_TOKEN=mynodetoken ./install.sh
 ```
 
 Note, take care to ensure you replace `myserver` with the IP or valid DNS of the server and replace `mynodetoken` with the node-token from the server.
```

Comment:
```suggestion
```

Since we're using `<VERSION>` in the new bit, maybe we can remove this line and update the command above it to 

```
INSTALL_K3S_SKIP_DOWNLOAD=true 
INSTALL_K3S_VERSION=<VERSION> 
K3S_URL=<SERVER>:6443 
K3S_TOKEN=<TOKEN> ./install.sh
```

Reply:
Updated placeholders in the command. Instead of removing this line I updated the phrasing instead since it provides context on the options used in the command.